companion/manifest.json

```diff
@@ -0,0 +1,26 @@
+{
+	"id": "metus-technology-ingest-controller",
+	"name": "Metus Technology Ingest Controller",
+	"shortname": "ingest-controller",
+	"description": "Ingest controller.",
+	"version": "0.0.0",
+	"license": "MIT",
+	"repository": "git+https://github.com/bitfocus/companion-module-metus-technology-ingest-controller.git",
+	"bugs": "https://github.com/bitfocus/companion-module-metus-technology-ingest-controller/issues",
```

Comment:
These URLs aren't correct, they 404. I don't know if there's a plan to rename it to match or not, but they're currently wrong.